ddsc/core/remotestore.py

```diff
@@ -370,3 +395,23 @@ def __init__(self, json_data):
 
     def __str__(self):
         return 'id:{} username:{} full_name:{}'.format(self.id, self.username, self.full_name)
+
+
+class RemoteAuthRole(object):
+    PROJECT_CONTEXT = "project"
+    SYSTEM_CONTEXT = "system"
```

Comment:
Is there a case where SYSTEM_CONTEXT would be used? or is it just here for documentation purposes?